static/js/common.js

```diff
@@ -136,3 +136,17 @@ export function adjust_mac_shortcuts(key_elem_class, require_cmd_style) {
         $(this).text(key_text);
     });
 }
+
+export function toggle_password(password_field, password_selector) {
+    $("body").on("click", password_selector, () => {
+        const password_id = $(password_field);
+
+        if (password_id.attr("type") === "password") {
+            password_id.attr("type", "text");
+        } else {
+            password_id.attr("type", "password");
+        }
+
+        $(password_selector).toggleClass("fa-eye fa-eye-slash");
+    });
+}
```

Comment:
I think most of this function could be extracted as a `toggle_password_visibility($password_field)`, and then have a wrapper function that also registers that as the click handler named something that'd be called as `common.setup_password_visibility_toggler("#ldap-password")`. -- that'd be a much nicer interface.

Also `password_id` is a confusing name for a jQuery object -- try something like `$password_field`.